website/content/docs/concepts/workers.mdx

```diff
@@ -0,0 +1,94 @@
+bucket](/boundary/docs/concepts/domain-model/storage-buckets) for the recording.
+
+## Tags
+In the multi-datacenter and multi-cloud operating models, patterns of dividing up controllers, workers, and targets into appropriate regions or networks is often 
+desired to reduce latency or comply with security standards. Workers can be assigned [tags](/boundary/tutorials/worker-management/target-aware-workers) that Boundary 
+can [filter](/boundary/docs/concepts/filtering/worker-tags) through, to find the appropriate worker to use for a session. For example, Boundary could filter to workers 
+with tag “A”, to connect to targets in “Network A”.
+
+![IAM Example Structure](/img/worker-tags.png)
+
+## Multi-hop sessions <sup>HCP/ENT</sup>
+Most organizations want to provide access to infrastructure without exposing private networks. Many organizations also have complex network topologies requiring 
+inbound traffic to route through multiple network enclaves in order to reach the target system. 
+[Multi-hop](/boundary/docs/configuration/worker#multi-hop-worker-capabilities-hcp-ent) sessions allow two or more workers to be chained together 
+across multiple networks to form reverse proxy connections between the user and the target, even in complex networks with strict outbound-only policies.
+
+In multi-hop scenarios, there are typically 3 types of workers:
+1. **Ingress worker** - An ingress worker is a worker that is accessible by the client. The client will initiate the connection to the ingress worker.
+2. **Intermediary worker** - An (optional) intermediary worker sits between ingress and egress workers as part of a multi-hop chain.
+3. **Egress worker** - An egress worker is a worker that can access the target. The egress worker initiates reverse proxy connections to intermediary or ingress workers.
+
+<Tip>
+“Ingress”, “intermediary”, and “egress” are general ways to describe how the respective worker interfaces with resources, and a worker can be more than 1 of those 
+at a time. For example in the diagram below, the intermediary worker is also an egress worker since it can access a target. 
+</Tip>
+
+![IAM Example Structure](/img/concepts-multihop.png)
+
+Once the persistent connection chain is established between the workers, when the user requests to connect to a target host, they are automatically proxied from:
+1. Boundary client to ingress worker
+2. Ingress worker to intermediary worker (where applicable)
+3. Ingress worker to egress worker
+4. Egress worker to desired target
+
+## Deployment
+Workers are services that can run on a container or virtual machine, and should be deployed strategically within networks to provide access to targets. In 
+all editions of Boundary, workers are fully self-managed and can be deployed anywhere. In HCP Boundary, 3 HCP-managed workers are automatically deployed with the cluster. 
```

Comment:
I would drop "3 HCP-managed" and just keep it as:
**_In HCP Boundary, HCP-managed workers are automatically deployed..._**

Reply:
Will change